Gemfile.lock

```diff
@@ -504,3 +516,6 @@ DEPENDENCIES
   xapian-full-alaveteli (~> 1.2.21.1)
   xml-simple (~> 1.1.0)
   zip (~> 2.0.0)
+
+BUNDLED WITH
```

Comment:
would prefer not to have the bundler version checked in

Reply:
ugh yeah